.github/workflows/build-and-publish-master.yaml

```diff
@@ -25,8 +45,26 @@ jobs:
         run: sbt Compile/compile Test/compile IntegrationTest/compile
       - name: Scala Test
         run: sbt "set ThisBuild/coverageEnabled := true" test IntegrationTest/test coverageAggregate
-      - name: DataflowCheck
-        run: sbt "hca-transformation-pipeline/run --inputPrefix=gs://broad-dsp-monster-hca-dev-test-storage/inputs/Success --outputPrefix=gs://broad-dsp-monster-hca-dev-test-storage/outputs-to-overwrite --runner=DataflowRunner --project=broad-dsp-monster-hca-dev --region=us-central1 --tempLocation=gs://broad-dsp-monster-hca-dev-temp-storage/dataflow"
+      - name: Run Dataflow against test data
+        env:
+          GITHUB_SHA: ${{github.sha}}
+        run: >
+          sbt "hca-transformation-pipeline/run
+                 --inputPrefix=gs://broad-dsp-monster-hca-dev-test-storage/integration/ebi_small/test_data
+                 --outputPrefix=gs://broad-dsp-monster-hca-dev-temp-storage/integration/output_${GITHUB_SHA}
+                 --runner=DataflowRunner
+                 --project=broad-dsp-monster-hca-dev
+                 --region=us-central1
+                 --tempLocation=gs://broad-dsp-monster-hca-dev-temp-storage/dataflow
+                 --workerMachineType=n1-standard-4
+                 --numWorkers=16
```

Comment:
In my testing, this got us a good speedup (from ~20 minutes -> 7ish against our previous "test dataset"), and is not a large cost delta.